### PR TITLE
Create DateUtils Class

### PR DESCRIPTION
**Pull request by kyhjonathan**
Created a new class that deals with the Time API

### CODE REVIEW

**Review by NicholasTanYY — COMMENTED** [review]
LGTM

**Review by clarencepohh — COMMENTED** [review]
LGTM! We could explore merging this class with WeekView for time-related functions. Let's settle dependency issues when all the important classes are created. 👍

File: src/main/java/Time/DateUtils.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package Time;
```
Comment (Mahesh1772) [inline src/main/java/Time/DateUtils.java:1]: Clear and concise implementation. Lets merge it later in our call!

**Review by Mahesh1772 — COMMENTED** [review]
Good code

**Pull request merged**